### PR TITLE
testing WWWServer

### PR DESCRIPTION
**Pull request by olivierteboul**

### CODE REVIEW

File: icubam/www/handlers/update.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def get_icu_data_by_id(self, icu_id, def_val=0):
     input_data = self.token_encoder.decode(user_token)
 
     if input_data is None:
-      return self.redirect('/error')
+      return self.set_status(400)
```
Comment (VincentRouvreau) [inline icubam/www/handlers/update.py]: Wouldn't it be better to return 404 ? Be sure that returning 400 is not a way to tell hackers "try again".

File: icubam/www/handlers/update.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def get_icu_data_by_id(self, icu_id, def_val=0):
 
     except Exception as e:
       logging.error(e)
-      return self.redirect('/error')
+      return self.set_status(400)
```
Comment (VincentRouvreau) [inline icubam/www/handlers/update.py]: Wouldn't it be better to return 404 ? Be sure that returning 400 is not a way to tell hackers "try again".

File: icubam/www/handlers/update.py (hunk 1 above)
Comment (olivierteboul) [reply icubam/www/handlers/update.py]: good point ! doing it now

**Comment by olivierteboul** [comment]
Thanks Vincent for the comments, very good point

**Pull request merged**